backend/users/migrations/0001_initial.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 3.0.2 on 2020-01-09 16:42
```

Comment:
Je propose de donner des noms explicites à nos fichiers de migrations pour qu'on s'y retrouve, par example `add_user_model.py` (tu peux le faire directement en ligne de commande a moment de créer le fichier d'après https://stackoverflow.com/questions/30425912/is-it-safe-to-rename-django-migrations-file)